pyfar/dsp/dsp.py

```diff
@@ -223,3 +224,75 @@ def spectrogram(signal, dB=True, log_prefix=20, log_reference=1,
     times -= times[0]
 
     return frequencies, times, spectrogram
+
+
+def deconv(measurement, excitation, **kwargs):
```

Comment:
What would be the best name for this?
- devonv
- deconvolution
- spectral_deconvolution